### PR TITLE
 Added some explanations to the tutorial for risingbubble.jl

### PR DESCRIPTION
**Pull request by smarras79**
# Description

Added:

```
* #    Top and Bottom boundaries:
* #    Lateral boundaries 
```
in `# 2) Boundaries `

and how to change the time integrator from multi-rate to single rate
```
*  # If the user prefers a single rate explicit time integrator,                                                                                        
    # the ode_solver above can be replaced with:                                                                                                         
    ode_solver = ClimateMachine.ExplicitSolverType(
        solver_method = LSRK144NiegemannDiehlBusch,
    )
```

* I would like to add a list of options for explicit `solver_method` but I don't know which ones are currently working and what type of parameters they would require. It would be good to add this though.

* It would also be good to update the tutorial to show how to modify the lateral boundary conditions if we don't want them to be periodic.

I have

- [ ] Written and run all necessary tests with CLIMA by including `tests/runtests.jl`
- [ ] Followed all necessary [style guidelines](https://CliMA.github.io/CLIMA/latest/CodingConventions.html) and run `julia .dev/climaformat.jl .`
- [ ] Updated the documentation to reflect changes from this PR.

# For review by CLIMA developers

- [ ] There are no open pull requests for this already
- [ ] CLIMA developers with relevant expertise have been assigned to review this submission
- [ ] The code conforms to the [style guidelines](https://CliMA.github.io/CLIMA/latest/CodingConventions.html) and has consistent naming conventions. `julia .dev/format.jl` has been run in a separate commit.
- [ ] This code does what it is technically intended to do (all numerics make sense physically and/or computationally)

### CODE REVIEW

File: tutorials/Atmos/risingbubble.jl [hunk 1]
```diff
@@ -213,6 +215,11 @@ function config_risingbubble(FT, N, resolution, xmax, ymax, zmax)
         fast_method = LSRK144NiegemannDiehlBusch,
         timestep_ratio = 10,
     )
+    # If the user prefers a single rate explicit time integrator,
+    # the ode_solver above can be replaced with:
+    ode_solver = ClimateMachine.ExplicitSolverType(
+        solver_method = LSRK144NiegemannDiehlBusch,
+    )
```
Comment (kpamnany) [inline tutorials/Atmos/risingbubble.jl:222]: This overwrites the previous `ode_solver` definition -- maybe you meant to have it in a comment?

Comment (akshaysridhar) [reply tutorials/Atmos/risingbubble.jl:222]: Right - this should either be a comment in here, or point to the driver configuration setup that allows a user to choose from those options.

Comment (smarras79) [reply tutorials/Atmos/risingbubble.jl:226]: I like @akshaysridhar idea

File: tutorials/Atmos/risingbubble.jl [hunk 2]
```diff
@@ -13,10 +13,12 @@
 # ## Description of experiment
 # 1) Dry Rising Bubble (circular potential temperature perturbation)
 # 2) Boundaries
-#    - `Impenetrable(FreeSlip())` - no momentum flux, no mass flux through
+#    Top and Bottom boundaries:
+#    - `Impenetrable(FreeSlip())` - Top and bottom: no momentum flux, no mass flux through
 #      walls.
 #    - `Impermeable()` - non-porous walls, i.e. no diffusive fluxes through
 #       walls.
+#    Lateral boundaries
```
Comment (akshaysridhar) [inline tutorials/Atmos/risingbubble.jl:21]: If we emphasize that these are default boundary conditions, the line below about Applying boundary conditions(in the scope of tutorial note block) can be removed.

**Comment by smarras79** [comment]
bors try

File: tutorials/Atmos/risingbubble.jl (hunk 1 above)
Comment (akshaysridhar) [reply tutorials/Atmos/risingbubble.jl:217]: MRRK is commented here, for the tutorial, fine with me

**Comment by smarras79** [comment]
@akshaysridhar please squash your commit and push again so that I can merge. Thanks

File: tutorials/Atmos/risingbubble.jl [hunk 3]
```diff
@@ -298,15 +309,20 @@ function main()
...
-    zmax = FT(2500)
+    xmax = FT(10000)
+    ymax = FT(500)
+    zmax = FT(10000)
     t0 = FT(0)
     timeend = FT(1000)
-    CFL = FT(20)
+
+    #Use up to 20 if ode_solver is the multi-rate LRRK144
+    CFL = FT(15)
```
Comment (akshaysridhar) [inline tutorials/Atmos/risingbubble.jl:322]: Needs to be commented since the MRRK is an alternate suggested time stepper.
